src/config/layers.json

```diff
@@ -697,7 +697,9 @@
   "avg_temperature_groundstations": {
     "title": "10-day average temperature",
     "type": "groundstation",
-    "data": "https://mng.prism.services/temp/Temperature?beginDateTime=2020-05-01&endDateTime=2020-05-01",
+    "data": "https://mng.prism.services/temp/Temperature",
```

Comment:
Let's try to avoid being too specific for this particular API. Can we just use `https://mng.prism.services/temp/Temperature?beginDateTime=2000-01-01&endDateTime=2023-12-21` for now?